lcli/src/eth1_genesis.rs

```diff
@@ -17,29 +17,19 @@ pub fn run<T: EthSpec>(
     testnet_dir: PathBuf,
     matches: &ArgMatches<'_>,
 ) -> Result<(), String> {
-    let endpoints = matches
-        .value_of("eth1-endpoint")
-        .map(|e| {
-            warn!("The --eth1-endpoint flag is deprecated. Please use --eth1-endpoints instead");
-            vec![String::from(e)]
-        })
-        .or_else(|| {
-            matches
-                .value_of("eth1-endpoints")
-                .map(|s| s.split(',').map(String::from).collect())
-        });
+    let endpoint = matches.value_of("eth1-endpoint").map(|e| {
```

Comment:
It seems like the logic here would ignore `--eth1-endpoints` . Is the goal to switch this to use `--execution-endpoint`?